### PR TITLE
Werewolves rejoice! (removes silver from the dispenser).

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request
This PR removes silver from the chem dispenser and synthesizer.

## Why It's Good For The Game
Silver and gold are in theory precious metals. However they don't feel very precious aboard space station 13  due to a glut of these elements.
By increasing demand for silver bars slightly, we can make them more precious, this also makes reagents made using the silver reagent; mainly formaldehyde and teslium more precious.

It also makes no sense to me why NT would put effort into mining silver when they can make infinite amounts of silver powder on the station using their chem dispensers.

I think this PR helps move the game in the right direction in the areas of scarcity and material balance.

## Changelog
:cl:
del: Removed silver from the chemical dispenser and synthesizer.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
can you add a reaction to turn silver reagent into silver bars then 😳

**Comment by Krysonism** [comment]
> 
> 
> can you add a reaction to turn silver reagent into silver bars then flushed

Sure

**Comment by FloranOtten** [comment]
Why are you trying to increase the price of core cobbychems? If there's no mining, you can no longer make lenturi. Nor silver sulfadizine. Why?

**Comment by necromanceranne** [comment]
Let's see what gets made really hard to make with this:

- Lenturi
- Teslium
- Formaldehyde

Lenturi is kinda bad anyway, Teslium is debatably useful and Formaldehyde is a really fundamental chem for virology and paramedics, and part of the recipe for Pentetic Acid, one of our remaining medicines that will without drawbacks purge toxins and reagents. It's also one of the most used chems in the recipe for Penthrite, which is our heart stabilization med.

The formaldehyde hit is probably the biggest of the lot, used in Pentaerythritol (Penthrite and TaTP), and Acetaldehyde (used to make Pentaerythritol). Formaldehyde also makes Heparin.

Figured I'd let everyone know what's being affected here. I can't think of anything else silver reagent might be important for to bring it up.

**Comment by Krysonism** [comment]
> 
> 
> Let's see what gets made really hard to make with this:
> 
>     * Lenturi
> 
>     * Teslium
> 
>     * Formaldehyde
> 
> 
> Lenturi is kinda bad anyway, Teslium is debatably useful and Formaldehyde is a really fundamental chem for virology and paramedics, and part of the recipe for Pentetic Acid, one of our remaining medicines that will without drawbacks purge toxins and reagents. It's also one of the most used chems in the recipe for Penthrite, which is our heart stabilization med.
> 
> The formaldehyde hit is probably the biggest of the lot, used in Pentaerythritol (Penthrite and TaTP), and Acetaldehyde (used to make Pentaerythritol). Formaldehyde also makes Heparin.
> 
> Figured I'd let everyone know what's being affected here. I can't think of anything else silver reagent might be important for to bring it up. This seems like hitting at specifically a core chem to the creation of high explosives in the form of TaTP and Penthrite.

If mining exists they will provide the station with silver bars. 

In a pinch, a silver coin is enough for 12 doses of formaldehyde.

**Comment by Krysonism** [comment]
> 
> 
> Why are you trying to increase the price of core cobbychems? If there's no mining, you can no longer make lenturi. Nor silver sulfadizine. Why?

Silver sulfazidine has not existed in a long, long time. So i am going to disregard this comment as uninformed.

**Comment by CRITAWAKETS** [comment]
this literally  just adds tedium to chemistry for no reason at all but "i dont like it"

troll pr/10

**Comment by Krysonism** [comment]
> 
> 
> this literally just adds tedium to chemistry for no reason at all but "i dont like it"
> 
> troll pr/10

read the PR description.

**Comment by anconfuzedrock** [comment]
this pr wasn't made based on the balance of what the chemical can actually make, it's made on the most ridiculous balance of "but space silver rare?" and your logic of "mining will probably bring in silver" doesn't mean that it's harder to make these chems, it means it's more inconsistent- sometimes it will be impossible to make these chemicals through absolutely no fault of the chemist. If you're really salty about space silver being rare, just... rename the reagent to not silver. "but space silver rare?" is the dumbest fluff reason I've ever heard and at that point you should be removing a lot of free shit besides this.

**Comment by antropod** [comment]
Are you making PRs to remove all reagents from dispenser? Sugar, silver, what else. Or is this a troll PR to see how mad can people be? I don't understand.

**Comment by Rustybuckets6601** [comment]
Why not just do what they did with stable plasma and create a new regeant called liquid silver or something? It just seems like unnecessary tedium

**Comment by antropod** [comment]
There is no silver solidification recipe like with gold and uranium. There is no silver producing plant. Stop breaking this game.

**Comment by itseasytosee** [comment]
its a chem dispenser. It breaks the law of energy conservation already.

**Comment by Krysonism** [comment]
>sometimes it will be impossible to make these chemicals through absolutely no fault of the chemist. 

Sometimes it will be impossible to build teleportation machinery because miners did not bring in any BS crystals.  That is a good thing.
If you need hundreds of u of teslium for some reason and all miners are dead, there is always public mining.

**Comment by antropod** [comment]
Another PR that solves no issue, just fucks up chemistry.
It breaks recipes not only for
List of chemicals affected:
`Lenturi
Teslium
Formaldehyde
Pentetic acid
Pentaerythritol
Acetaldehyde
Penthrite
TaTP
Medicated Suture
`
Pentetic acid and formaldehyde are most used of them. No more factories for you chemists, because silver was removed for no good reason, just because "spess silver expensive"

**Comment by Krysonism** [comment]
> 
> 
> Another PR that solves no issue, just fucks up chemistry.
> It breaks recipes not only for
> List of chemicals affected:
> 
> ```
> Teslium
> Formaldehyde
> Pentetic acid
> Pentaerythritol
> Acetaldehyde
> Penthrite
> TaTP
> Medicated Suture
> ```
> 
> Pentetic acid and formaldehyde are most used of them. No more factories for you chemists, because silver was removed for no good reason, just because "spess silver expensive"

One silver coin from maint can make 72u pentetic.

You can still make factories, you have a grinder module in the plumbing constructor for a reason.

**Comment by antropod** [comment]
How often do you see factories with grinders? People just won't bother doing that if its not fully automatic.

**Comment by Krysonism** [comment]
> How often do you see factories with grinders? People just won't bother doing that if its not fully automatic.

If they are too lazy to toss a stack of silver(1000u) into the grinder once per round, that is their loss.

> Effectively you are removing part of chemistry without replacement FNR

I am not removing any part of chemistry, you can still get silver by grinding silver bars, like god intended.

**Comment by Rohesie** [comment]
This seems good for the long-term health of the game.

**Comment by antropod** [comment]
Yes, that's why I said "Effectively". You are not removing it, but making it hard enough for chemists to stop doing it.

**Comment by Learner10** [comment]
Aside from the huge blow to pentetic acid and hit on formaldehyde, I myself don't see this as too awful a change. If there are no miners, it's not like the chemist can't just go down to Lavaland to get it. At this point, someone will respond with "he shouldn't have to go down there", whereas I'll just slap this factoid back down at them; he can probably look for lavaland chems while he's down there as well. 

You don't need much silver to get some units, and pentetic acid is considerably one of the most valuable C3 chems for how useful it is. Like... stupid useful in that it completely invalidates even using anything else. The change would mean that trying to get it requires another department (or you) to do some work to get the material for it, and it's not like syriniver where it needs a bs rare chem to make. It's just silver. One of the less needed materials available. Hell, even if there aren't any miners, you could just steal the surgery tables that no one also uses.

Think of it like this... libital gets used all the time, as does aiuri, but multiver does not. Most people usually ignore all of the C2 chems for toxins, and instead try to jump right to Pentetic Acid. No hot seiver, no syriniver, no multiver, and no calomel. Hell, even antihol is technically a free toxin damage remover, is made with multiver that no one was probably going to use, and even that's somehow less relevent then all of the above.

Hell... most people forget that cryo is one of the most free methods available to remove poison damage, yet rarely will you see someone using that as well...

Pentetic's too strong, trivialize it a little. It also means that a smart traitor can try to steal all the silver before unleashing his bioweapons and not worry about an endless reserve of countering medicine sitting around medical.

**Comment by itseasytosee** [comment]
If you wanted a good compromise just give em a bit of silver to work with in the beginning, kinda like how they have plasma. I'd be happy with that so I don't have to scrub maintenance for silver coins.

**Comment by Learner10** [comment]
> 
> 
> If you wanted a good compromise just give em a bit of silver to work with in the beginning, kinda like how they have plasma. I'd be happy with that so I don't have to scrub maintenance for silver coins.

Or steal the surgery tables that no one uses, with the multiple free set of tools that you also start the round with.

**Comment by ATH1909** [comment]
Aw, but I _like_ my plumbing-based tesla coils!

**Comment by ATH1909** [comment]
If you're making penthrite require a lavaland chem (silver) anyway, can you remove wittel from its recipe?

**Comment by Krysonism** [comment]
syriniver and new multiver are extremely good.

And pentetic is only like 5% silver, so it doesn't even take that much silver to produce pentetic.

**Comment by ExcessiveUseOfCobblestone** [comment]
this is fine /w silver solidification.

In general i am for devaluing the chem dispenser and synthesizers, see https://hackmd.io/@tgstation/r1Vt1CNoU#Desired-State-of-Reagent-Pipeline

**Comment by Losenis** [comment]
You're nerfing the availability of Formaldehyde and Pentetic acid, both of which see usage in medbay by preventing corpse rot and dealing with toxins and rads. This would effectively turn radiation issues into a nightmare if pentetic acid supply is not guaranteed due to

A) Just having none due to no silver and bad maint RNG, assuming the chemist doesn't get yeeted in maint by whatever might be in there since not every round is a greenshift.
B) Extremely short supply of it by grabbing coins and the sort, which already puts it in competition with formaldehyde by how it's used in the recipe. It's going to run out extremely fast if the rads are so high it starts jumping onto other people, as well as pentetic acid also competing in usage for normal toxin damage and reagent purging.
C) Having to go through the tedium of grabbing silver from the lathe and grinding it and keeping it around to handle physically rather than just having it in the dispenser with the other reagents. This also runs the workplace risk of just accidentally spilling it, even by simply trying to put it on the table with the wrong intent, unless they go through more tedium of making plastic bottles to cap and uncap with every usage.
D) Keeping a pile of silver in chem to grind as needed, which means silver taken out from the overall stock of the station. With how the ORM no longer is upgradeable, running out of the stuff is an actual issue unless there's enough miners or someone who actually enjoys the strip mining aspect of mining.

Silver in liquid form can't even be turned into solid bars, contrary to gold which isn't in the dispenser for that specific reason of internal inbalance. The silver in the dispensers is essentially a completely unrelated reagent to the silver bars other than how you grind silver elements for it. You could just rename the dispenser silver to another reagent and add an actual silver reagent for some special chem recipes rather than nerfing large aspects of chemistry over how rare one believes a reagent that happens to share a name with a precious metal, regardless of what balance it already has without adding something in to compensate. Medbay already has a big history of nerfs, if they get any more it should be over balance concerns rather than lore.

**Comment by gryffinp** [comment]
Is this a satire about the sugar removal PR?

**Comment by Yenwodyah** [comment]
If your end goal is "gut chemistry" can you rip the name-brand adhesive medical strip off at once instead of dragging it out?

**Comment by nianjiilical** [comment]
if this goes through i would very much like to see formaldehyde's recipe changed since its borderline a necessary chem for big medical situations

**Comment by Krysonism** [comment]
>running out of the stuff is an actual issue unless there's enough miners 

Perfect!

>This would effectively turn radiation issues into a nightmare if pentetic acid supply is not guaranteed

Pentetic is not even that good at treating rads. Cold seiver is way better.
In fact, while pentetic is the best all round toxins chems, it does not excell on any area. Multiver and calomel purges faster and more selective, syriniver heals toxins faster and cold seiver purges rads faster.

Pentetic is essentially a luxury chem and should be treated as such. In 90% of rounds you will have no issue obtaining sufficient silver.

**Comment by Winterous** [comment]
Aah yes, because preserving the stack of corpses lying around doesn't happen rarely enough!
This is a terrible idea until the infrastructure surrounding chemlab is changed to use reagent silos, like is planned.

**Comment by anconfuzedrock** [comment]
If the point is scarcity, then why the FUCK did you pick silver? SILVER?!! This is used in a completely arbitrary list of recipes. You could've throwed a dart at your computer screen with the chem dispenser open to pick which one to change. Fuck, even swap the recipes silver is used for with that of another reagent. Just pick a chemical that makes SENSE to restrict. (This is still an incredibly stupid idea though for the reason I said earlier of mining and chemistry having literally nothing to do with each other. this isn't interdepartmental cooperation, chemists could get their silver without ever seeing a miner and them asking miners to get silver isn't going to do anything except annoy miners. This is also an incredibly stupid idea if this is one of the only chemicals from the dispenser to restrict.)

**Comment by Losenis** [comment]
> Perfect!

It's not. Medbay's already been getting nerfed, this is nerfing it for the sake of fluff rather than imbalance. There hasn't been balance issues sited with silver as a chem dispenser reagent, only that it should not be there over just happening to share the name with a precious metal.

> Pentetic is essentially a luxury chem and should be treated as such. In 90% of rounds you will have no issue obtaining sufficient silver.

Doesn't matter. It's not a balance issue. Let it be handled by its own PR if it is, this one is just causing scarcity for the sake of scarcity.

> This would not increase material and reagents scarcity which is the point of this PR.
Imma be real with you chief. If the reagent was named anything else, this PR would've never even been thought of. The balance issues cited afterwards in the comments sound more like "The consequences are a sacrifice I'm willing to make" than actual concerns, since they were conveniently exempt in the original post despite fiddling with a reagent that is used in multiple recipes, ignoring it because "oh they can get it from elsewhere". Yeah, ghetto chem is a thing, you can get reagents from many sources without a chem dispenser, but it's medbay, they're the department solely dedicated to making and administering medicines, they're meant to have those things readily available. If you disagree, then that's a PR to be handled elsewhere by changing up Chemistry and adding the infrastructure required to make them rely less on chem dispensers, not just removing stuff and assuming nothing would break. balance will magically remain the same.

This effectively forces ghetto chem on actual chem for that reagent, and I can see this slowly starting to cover iron, silicon (glass), welding fuel, stable plasma, water, and other chems that have the same argument to be made for when it comes to "just get it from somewhere else bro". It doesn't add fun, it adds tedium, and if this change isn't meant to benefit the player through fun, through balance, through gameplay, then what's the point? You'll just see chems skipping anything that needs silver until it's absolutely required and fuck around with something else like meth/mannitol pills or whatever else if it's a calm shift and there isn't much else to do. I know I would, because I wouldn't be signing up as chemist just to fuck around as miner or maintdiving assistant.

**Review by Rohesie — CHANGES_REQUESTED** [review]
As Cobby said, add silver solidification and this is fine to go.

**Comment by Winterous** [comment]
In no uncertain terms, this is one of the worst ideas I've ever seen. Until the infrastructure for dispensers drawing from a station reagent silo system is in place, there is no reasonable way to justify this.

The process of grinding reagents to use in your recipes is absolutely awful, slow, tedious, and a needlessly fiddly source of inventory management. For things like plasma and uranium it's ok, because only a few very specialised recipes use those things, not things you'll make multiple times a shift, most shifts, or which constitute a critical part of plumbing factories.

Shit like this will make chemists do their job even LESS, getting a few formaldehyde patches out to preserve the pile of corpses is something that rarely happens unprompted, it'll basically never be done with this change.
At the VERY VERY LEAST, you should add like 5 bars of silver to chemlabs on every map, like there are plasma bars; but you haven't done that, because this PR is incredibly lazy in addition to being incredibly stupid. You've decided to make this change for entirely superficial reasons without any prior considerations of the gameplay consequences or ways you might mitigate any negative effects.

**Comment by Learner10** [comment]
1 piece of material equals 20u of that substance in liquid form. Formaldehyde requires 1 unit of silver to produce 3 units of formaldehyde. 20 units of silver equals 60 units of formaldehyde. If you're a resource conservative paramedic or doctor, that's literally 60 corpses you can preserve (1u per corpse through eye dropper). If you're in a hurry, 60 units of formaldehyde is still at minimum 12 corpses preserved with a syringe full of pure formaldehyde (assuming one dose per corpse). If you add 60 units of water to your 60 units of formaldehyde, (aka literally watering down the formaldehyde like you would syriniver), that minimum number of 12 suddenly jumps to 24 corpses.

Oh yea, and medical usually has like 2-3 surgical beds, so that's 40 units of silver round start if you're THAT desperate.

If anyone's saying that the lack of chemical silver is going to be an issue, I've got no reason but to believe that you're either speaking a load of hock, or are using WAY too much formaldehyde on the bodies. The only valid complaint about this change is that you'll have to spend at most 30 seconds to walk to the medical lathe to grab silver, and then spend 5-15 seconds putting  the silver in a grinder to use it. 

Frankly, I'm concerned that the change is doing too little, save for corking the infinite pentetic acid meme generator.

**Comment by Winterous** [comment]
Or if you read what people are saying, you'd know that it's not just concern over availability, but how much of a pain in the ass it is to go get sheets, grind them, and then needing to juggle bottles of reagents to put them to use. All for literally no reason; this doesn't improve any balance, it doesn't make the experience more rewarding, it doesn't do anything except make the job more annoying to do; oh, and make it harder to cure viruses or deal with massive radiation hazards quickly.
Because radiation isn't hard enough to tackle already.

**Comment by Krysonism** [comment]
>make it harder to cure viruses or deal with massive radiation hazards quickly.
Because radiation isn't hard enough to tackle already.

As I've already discussed pentetic is already not the best option for treating radiation. Seiver is far superior and can be combined with potassium iodide.

Too many people treat pentetic like the default option.

>At the VERY VERY LEAST, you should add like 5 bars of silver to chemlabs on every map

No, that would be enough to last all round, that would defeat the purpose of this PR.

**Comment by Winterous** [comment]
Seiver doesn't heal the toxin damage you're taking from radiation though, just the radiation itself. If half the station is contaminated you're also going to want to be on the stuff 24/7 anyway, which means the heart damage will be an issue.

Also, your solution to "I cannot make these important chemicals at the start of the shift or if there are no miners" is REALLY to deconstruct the surgical tables? Why should you have to cannibalise your department to do your goddamn job in a timely manner?
Fine, it doesn't have to be FIVE sheets, just ONE sheet. Sure. Do you object to that pittance?

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -586,3 +586,13 @@
...
 	required_reagents = list(/datum/reagent/plasma_oxide = 1,/datum/reagent/stabilizing_agent = 1)
+
+/datum/chemical_reaction/silverolidification
+	required_reagents = list(/datum/reagent/silver = 20, /datum/reagent/carbon = 10)
+	required_temp = 630
+	mob_react = FALSE
+
+/datum/chemical_reaction/silversolidification/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
```
Comment (Rohesie) [inline code/modules/reagents/chemistry/recipes/others.dm:597]:
```suggestion
	for(var/i in 1 to created_volume)
```
The list loop is faster than the C loop, not to mention shorter and more intuitive to those not used to C.

**Comment by PKPenguin321** [comment]
seems it would be very, very easy to just make the recipes that use silver use a different reagent instead

**Comment by Winterous** [comment]
> 
> 
> seems it would be very, very easy to just make the recipes that use silver use a different reagent instead

On that note I looked up how formaldehyde is made out of curiosity, and the silver is a CATALYST, not a component; so silver shouldn't be consumed in making formaldehyde at all.

**Comment by Angustmeta** [comment]
Nah silver is not a catalyst in the formaldehyde recipe.

**Comment by antropod** [comment]
Every recipe where gold is used it is a catalyst.

**Comment by antropod** [comment]
@Rohesie 
2 line PR with no thought involved, which fucks up whole department gets merged?
He even misspelled `solidification` - this is how little fuck was given

**Comment by Winterous** [comment]
Did Rohesie have a fucking aneurysm or something? In what bizarre alternate reality could this PR be even remotely considered good enough to merge?

**Comment by Losenis** [comment]
So medbay keeps getting nerfed but people dropped the act that it was for balance?

**Pull request merged**